### PR TITLE
[UNTIL-10933]Barcode scanner tablet optimization

### PR DESCRIPTION
**Pull request by shekar-allam**

### CODE REVIEW

File: scan-engine/src/main/res/values/themes.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <style name="Theme.InputActivity" parent="android:Theme.Material.Light.NoActionBar">
```
Comment (djordjeh) [inline scan-engine/src/main/res/values/themes.xml:3]: It's not InputActivity and above you use BarcodeActivity.

File: scan-engine/src/main/java/de/tillhub/scanengine/theme/Theme.kt [hunk 2]
```diff
@@ -13,7 +13,7 @@ import androidx.compose.runtime.Composable
 import androidx.compose.ui.tooling.preview.Preview
 
 @Composable
-internal fun ScanEngineTheme(
+internal fun AppTheme(
```
Comment (djordjeh) [inline scan-engine/src/main/java/de/tillhub/scanengine/theme/Theme.kt:16]: Why did you change the name from ScanEngineTheme -> AppTheme ?

Comment (shekar-allam) [reply scan-engine/src/main/java/de/tillhub/scanengine/theme/Theme.kt]: Just to keep consistent with how we did in other apps but I reverted it as its not necessary.
